CMakeLists.txt

```diff
@@ -820,20 +816,34 @@ function(build_loadable_extension_directory NAME OUTPUT_DIRECTORY PARAMETERS)
       COMMAND emcc $<TARGET_FILE:${TARGET_NAME}>.lib -o $<TARGET_FILE:${TARGET_NAME}> -sSIDE_MODULE=1 -O3
       )
   endif()
+  add_custom_command(
+    TARGET ${TARGET_NAME}
+    POST_BUILD
+    COMMAND
+    printf "${DUCKDB_NORMALIZED_VERSION}" > duckdb_version_file
+    COMMAND
+    printf "${EXTENSION_VERSION}" > extension_version_file || ( echo "CMake appears not to propagate correctly version information for the extension, using empty string" && rm extension_version_file && touch extension_version_file )
```

Comment:
Just wondering now, does this work on windows?

Reply:
This was not working on windows, moved to echo_append